example/products.rb

```diff
@@ -58,6 +58,11 @@ def top_products(first:)
 end
 
 class ProductSchema < GraphQL::Schema
+  if Gem::Version.new(GraphQL::VERSION) < Gem::Version.new('1.12.0')
```

Comment:
All the other test schemas already have this code block, these two just happened to be missed 🤷‍♂️